File: src/main/java/ChatLoginController.java
import javafx.fxml.FXML;
import javafx.fxml.Initializable;
import javafx.scene.control.Button;
import javafx.scene.control.Label;
import javafx.scene.control.TextField;
import javafx.scene.layout.AnchorPane;

import java.net.URL;
import java.util.ResourceBundle;

public class ChatLoginController implements Initializable {
    @FXML
    protected AnchorPane ChatLogIn;

    @FXML
    protected TextField NameTF;

    @FXML
    protected TextField PasswordTF;

    @FXML
    protected Label NameTitleLabel;

    @FXML
    protected Label PasswordTitleLabel;

    @FXML
    protected Label TitleLabel;

    @FXML
    protected Button LoginBTN;

    @Override
    public void initialize(URL location, ResourceBundle resources) {

    }
}

File: src/main/scala/ChatLoginController.scala
import javafx.event.ActionEvent
import javafx.fxml.{FXML, Initializable}
import javafx.scene.control.{Button, Label, TextField}
import javafx.scene.layout.AnchorPane

import java.net.URL
import java.util.ResourceBundle


class ChatLoginController extends Initializable {
  @FXML protected var ChatLogIn: AnchorPane = _
  @FXML protected var LoginTF: TextField = _
  @FXML protected var PortTF: TextField = _
  @FXML protected var IPTF: TextField = _
  @FXML protected var LoginTitleLabel: Label = _
  @FXML protected var PortLabelTitle: Label = _
  @FXML protected var TitleLabel: Label = _
  @FXML protected var LoginBTN: Button = _

  @FXML def actionLoginBTN(Event: ActionEvent): Unit = {}


  override def initialize(location: URL, resources: ResourceBundle): Unit = {
  }
}
